lib/Template/Provider.pm

```diff
@@ -577,6 +577,8 @@ sub _load_compiled {
     # one needs to care about slash direction is when dealing
     # with Module::Name -> Module/Name.pm translation.
     my $fpath = File::Spec->rel2abs( $file );
+    $fpath =~ /^(.*)$/s;
```

Comment:
This is faster and less intrusive I think.

$fpath =~ tr///;

Reply:
Thanks, I don't speak Perl well. I created a new PR, see #262